_vendor/github.com/gohugoio/gohugoioTheme/layouts/partials/site-footer.html

```diff
@@ -17,9 +17,9 @@
           </ul>
 
           <ul class="center f6 list ma0 mv4 pa0 tc">
-            <li class="dib mr3"><a href="https://twitter.com/gohugoio" class="dim link light-gray pv2">@GoHugoIO</a></li>
-            <li class="dib mr3"><a href="https://twitter.com/spf13" class="dim link light-gray pv2">@spf13</a></li>
-            <li class="dib"><a href="https://twitter.com/bepsays" class="dim link light-gray pv2">@bepsays</a></li>
+            <li class="dib mr3"><a href="https://x.com/gohugoio" class="dim link light-gray pv2">@GoHugoIO</a></li>
```

Comment:
The theme is vendored. Please make these changes in the https://github.com/gohugoio/gohugoioTheme repository.